docs/refs.bib

```diff
@@ -30,6 +30,14 @@ @article{BorisBook1973
   author = {Jay P Boris and David L Book}
 }
 
+@book{durran2013numerical,
```

Comment:
I would also add the original citation, Zalesak 1979

Reply:
just updated the citation